pkg/process/checks/net.go

```diff
@@ -533,3 +598,47 @@ func getNetworkID(sysProbeClient *http.Client) (string, error) {
 	}
 	return networkID, err
 }
+
+func (c *ConnectionsCheck) checkCapacity() (bool, error) {
+	if c.sysprobeClient == nil {
+		return false, fmt.Errorf("system probe client is nil")
+	}
+
+	capacityCheckStart := time.Now()
+	url := sysprobeclient.ModuleURL(sysconfig.NetworkTracerModule, "/connections/check_capacity?client_id="+ProcessAgentClientID)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return false, fmt.Errorf("error creating capacity check request %s: %w", url, err)
+	}
+
+	log.Tracef("Checking connections capacity endpoint: %s", url)
```

Comment:
perhaps you should use `shouldLog` as if the log level is not trace (99% of the time), then this is a redundant cpu consumption